src/main/ps2/SDL_ps2_main.c

```diff
@@ -24,11 +26,23 @@ static void prepare_IOP()
    sbv_patch_disable_prefix_check();
 }
 
+static void init_drivers() {
+    init_fileXio_driver();
+    init_memcard_driver(true);
+}
+
 int main(int argc, char *argv[])
 {
+    int res;
     prepare_IOP();
+    init_drivers();
+
+    res = SDL_main(argc, argv);
 
-    return SDL_main(argc, argv);
+    deinit_memcard_driver(true);
```

Comment:
you made an `init_drivers` for init but doing inline de-init instead of a `deinit_drivers` ?

Reply:
Ok, let’s create a method for it as well